CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Quote framework names in OTHER_LDFLAGS
```

Comment:
nit: add two empty spaces after each entry `Quote framework names in OTHER_LDFLAGS  `